pkg/server/server.go

```diff
@@ -583,6 +598,19 @@ func (s *Server) Run(ctx context.Context) error {
 		return err
 	}
 
+	// add post start hook that starts all registered controllers.
+	if err := s.AddPostStartHook("kcp-start-openapiv3-server", func(srvcontext genericapiserver.PostStartHookContext) error {
```

Comment:
Comment needs updating and I would maybe call this hook `kcp-start-openapiv3-controller`. I don't think it's called the openapiv3-server anywhere.